assets/process-html.js

```diff
@@ -0,0 +1,22 @@
+var { exec } = require('child_process');
+var path = require("path");
+
+var glob = require("glob");
```

Comment:
Super minor but it seems you use a mixture of `"` and `'` I think you should pick one and stick with it.